Core/GraphQL/GraphQLFiles/sell_getFulfilmentPoints.graphql

```diff
@@ -0,0 +1,7 @@
+query getFulfilmentPoints($pageSize: Int!, $page: Int = 1, $filters: FulfilmentPointFilter = null) {
+  getFulfilmentPoints(page: $page, pageSize: $pageSize, filters: $filters) {
+    edges {
+      ...FragmentFulfilmentPoint
+  	}
+  }
```

Comment:
Do we need `nextPage` here?

Reply:
Doesn't look like we do currently, but will add it in incase we want to paginate in future